api/ceramic_cache/models.py

```diff
@@ -42,6 +47,14 @@ class StampType(IntEnum):
         default=StampType.V1, choices=[(tag.value, tag.name) for tag in StampType]
     )
 
+    compose_db_save_status = models.CharField(
+        max_length=10,
+        choices=ComposeDBSaveStatus.choices,
+        default=ComposeDBSaveStatus.PENDING,
```

Comment:
For existing records, would it not make more sense to have this attribute as `nullable`and set `null` as the default value?
If we set the default value as PENDING, then this will suggest that all existing stamps are pending to be written to composed, which is not true.